package.json

```diff
@@ -53,6 +57,7 @@
     "sass": "^1.49.0",
     "sass-loader": "^7.1.0",
     "style-loader": "^0.21.0",
+    "typescript": "^4.5.5",
```

Comment:
Where do you use TypeScript in this PR?

Note that MB uses Flow, not TypeScript.  We could bundle both TS and Flow types in this repo in the future, but TS should probably be a dev or peer dependency instead.

Reply:
Removed typescript. Flow seems good for our usage!